Geometry/MuonSimData/data/muonProdCuts/2021/v3/muonProdCuts.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0"?>
+<DDDefinition>
+ <SpecParSection label="ProductionCuts" eval="true">
+  <SpecPar name="Muon">
+   <PartSelector path="//MUON"/>
+   <Parameter name="CMSCutsRegion" value="Muon" eval="false"/>
+   <Parameter name="ProdCutsForGamma" value="1.0*cm"/>
+   <Parameter name="ProdCutsForElectrons" value="1.0*cm"/>
+   <Parameter name="ProdCutsForPositrons" value="1.0*cm"/>
+  </SpecPar>
+  <SpecPar name="MuonIron">
+   <PartSelector path="//MGNT"/>
+   <PartSelector path="//YBSepar.*"/>
+   <PartSelector path="//COI.*"/>
+   <PartSelector path="//COCF"/>
+   <PartSelector path="//YB.*"/>
+   <PartSelector path="//ZStopSpace"/>
+   <PartSelector path="//SupportTubeForHE.*"/>
+   <PartSelector path="//YN.*"/>
+   <PartSelector path="//YE1.*"/>
+   <PartSelector path="//YE2.*"/>
+   <PartSelector path="//YE3.*"/>
+   <!-- if you want to re-stage YE4, then remove the comments around this element   <PartSelector path="//YE4.*"/>   -->
+   <PartSelector path="//ShieldingME4.*"/>
+   <PartSelector path="//ShieldingBoronPoly.*"/>
+   <PartSelector path="//ME11SupportDisk"/>
+   <Parameter name="CMSCutsRegion" value="MuonIron" eval="false"/>
+   <Parameter name="ProdCutsForGamma" value="1.0*cm"/>
+   <Parameter name="ProdCutsForElectrons" value="1.0*mm"/>
+   <Parameter name="ProdCutsForPositrons" value="1.0*mm"/>
+  </SpecPar>
+  <SpecPar name="MuonChamber">
+   <PartSelector path="//MB.*N"/>
+   <PartSelector path="//MB.*P"/>
+   <PartSelector path="//ME.*Space"/>
+   <PartSelector path="//RR.*"/>
+   <PartSelector path="//GE.*"/>
+   <Parameter name="CMSCutsRegion" value="MuonChamber" eval="false"/>
+   <Parameter name="ProdCutsForGamma" value="1.0*mm"/>
+   <Parameter name="ProdCutsForElectrons" value="0.002*mm"/>
+   <Parameter name="ProdCutsForPositrons" value="0.002*mm"/>
+  </SpecPar>
+  <SpecPar name="MuonSensitive_DT-CSC">
+   <!-- 1 keV for gamma/e-, 1 MeV for e+ -->
+   <PartSelector path="//MB.*PhiGas"/>
+   <PartSelector path="//MB.*ZGas"/>
+   <PartSelector path="//ME.*ActiveGasVol"/>
+   <PartSelector path="//GHA.*"/>
+   <Parameter name="CMSCutsRegion" value="MuonSensitive_DT-CSC" eval="false"/>
+   <Parameter name="ProdCutsForGamma" value="1.0*cm"/>
+   <Parameter name="ProdCutsForElectrons" value="1.0*mm"/>
+   <Parameter name="ProdCutsForPositrons" value="1.0*cm"/>
+  </SpecPar>
+  <SpecPar name="MuonSensitive_RPC">
+   <!-- 10 keV for gamma/e-, 1 MeV for e+ -->
+   <PartSelector path="//MB.*RPC_.*Gas.*"/>
+   <!--    <PartSelector path="//REA.*"/>  -->
+   <PartSelector path="//REB.*"/>
+   <PartSelector path="//REC.*"/>
+   <!--    <PartSelector path="//RED.*"/>  -->
+   <PartSelector path="//REE.*"/>
+   <PartSelector path="//REF.*"/>
+   <!--    <PartSelector path="//REG.*"/>  -->
+   <!--    <PartSelector path="//REH.*"/>  -->
+   <Parameter name="CMSCutsRegion" value="MuonSensitive_RPC" eval="false"/>
+   <Parameter name="ProdCutsForGamma" value="1.0*cm"/>
```

Comment:
The original value for this line was 8.5 m, and you are changing it to 1.0 cm. How is this change related to the demo GEM chamber? What effect will it have to make such a large change in value?

Reply:
This is done in consultation with Vladimir - 8.5m means no production of photons at all which is not a reliable cut off for Geant4 (might be OK for Geant3).